api/lib/session/bloc/session_bloc.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:convert';
+
+import 'package:broadcast_bloc/broadcast_bloc.dart';
+import 'package:equatable/equatable.dart';
+import 'package:player_repository/player_repository.dart';
+
+part 'session_event.dart';
+
+part 'session_state.dart';
+
+class SessionBloc extends BroadcastBloc<SessionEvent, SessionState> {
+  SessionBloc() : super(const SessionState()) {
+    on<OnPlayerAdded>(_onPlayerAdded);
+    on<OnPointsAdded>(_onAddPoints);
```

Comment:
OnPlayerDisconnected will remove the player from players list

Reply:
@noorbhatia  done with issues :heavy_check_mark: